eth2/state_processing/src/per_block_processing.rs

```diff
@@ -469,7 +469,7 @@ pub fn process_transfers<T: EthSpec>(
     spec: &ChainSpec,
 ) -> Result<(), Error> {
     verify!(
-        transfers.len() as u64 <= spec.max_transfers,
+        transfers.len() as u64 >= spec.max_transfers,
```

Comment:
Did you mean to commit this?

Reply:
Oh I did not! Good catch, thanks

Reply:
Np, saw it in my notifications and was like hmmm 🤔